purity/gc/gc.go

```diff
@@ -48,12 +48,14 @@ func Run(url, instanceName, tokenFile string, tls bool, minAge time.Duration, re
 		return err
 	} else if err := gc.LoadAllBlobs(); err != nil {
 		return err
+	} else if err := gc.RemoveActionResults(); err != nil {
+		return err
 	} else if err := gc.MarkReferencedBlobs(); err != nil {
 		return err
 	} else if err := gc.RemoveBlobs(); err != nil {
 		return err
 	} else if err := gc.RemoveBrokenBlobs(); err != nil {
-		return err
+		log.Notice("Failed to remove broken blobs: %v", err)
```

Comment:
The other functions are best effort/log errors rather than return them. RemoveBrokenBlobs is used by Clean also and needs to return an error. I put a log line here so replicate is always called regardless of the outcome of RemoveBrokenBlobs

Reply:
`ReplicateBlobs` wont be called if there's an error though :sweat_smile:

Reply:
Haha oops. I've set it to just return the error